scripts/transpile.js

```diff
@@ -0,0 +1,19 @@
+const { transpileFiles } = require('@asyncapi/generator-react-sdk');
+const path = require('path')
+
+async function main(){
```

Comment:
I don't think it's needed to wrap it like this in "main" function cause transpileTemplate is a wrapper already

Reply:
done 👍